query-connector/src/app/queryBuilding/buildFromTemplates/page.tsx

```diff
@@ -38,6 +47,72 @@ export default function QueryTemplateSelection() {
     queryName: false,
     selectedConditions: false,
   });
+  const [conditionValueSets, setConditionValueSets] =
```

Comment:
this holds the complete ValueSet data in state, whereas `selectedConditions` is only the Name/Include attributes